module/onboarding/Ping/README.md

```diff
@@ -0,0 +1,16 @@
+# Ping
```

Comment:
Have a go looking at other readme files in the repository and fill this and the Pong `README.md` in. These readme files are actually used to generate some of the pages on NUbook!